src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -76,7 +76,8 @@ const Sidebar = React.memo(({ type, filterOptions, triggerHistory, hasMore, setP
             return HISTORY_LABEL.ENVIRONMENT
         }
     }
-
+    filterOptions.sort((a,b)=>(a.label > b.label)?1:-1)
```

Comment:
What if filteredOption doesn't have any value? Is it handled for the empty case ? otherwise sorting in filterOptions will break